tcclient/http.go

```diff
@@ -13,7 +13,7 @@ import (
 	"reflect"
 
 	"github.com/taskcluster/httpbackoff"
-	"github.com/tent/hawk-go"
+	hawk "github.com/tent/hawk-go"
```

Comment:
Thank you Greg! I had never considered before that for a library, the repository name is allowed to differ from the directory name it is contained in, but now you highlight this, it makes total sense!